src/server.jsx

```diff
@@ -37,9 +56,31 @@ export default async function ({ req, res, context }) {
     return res.redirect(context.url);
   }
 
-  // console.log('###', context);
+  const promises = routes.reduce((matches, route) => {
+    const match = matchPath(req.url, route);
+    if (match) {
+      matches.push(
+        route.component.fetchData ?
+          route.component.fetchData({ match, dispatch: store.dispatch }) : Promise.resolve(null),
+      );
+    }
+    return matches;
+  }, []);
```

Comment:
We are prefetching data here from the server, in this case, we are preloading the images before the app is sent